SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security
+
```

Comment:
I blended the Github and Electron README's together. 

We could alternately put this directly in our `.github` community folder to have a policy that applies to all repos in the `vega` org, but I figured it would be safe and meaningful to start with `vega/vega` (since this is the core browser runtime that every other package is importing).

Reply:
Yeah, makes sense to have one for Vega specifically. We could also link to the org one for more info so we only have to maintain one place.